### PR TITLE
Update the live ebuild from the Gentoo tree

### PR DESCRIPTION
**Pull request by williamh**
- Move to eapi 6
- use symbolic links for docker-containerd and docker-runc instead of
  patching the sources

### CODE REVIEW

File: app-emulation/docker/docker-9999.ebuild [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 if [[ ${PV} = *9999* ]]; then
 	# Docker cannot be fetched via "go get", thanks to autogenerated code
 	EGIT_REPO_URI="https://${EGO_PN}.git"
 	EGIT_CHECKOUT_DIR="${WORKDIR}/${P}/src/${EGO_PN}"
-	inherit golang-base git-r3
+	inherit git-r3
 else
 	MY_PV="${PV/_/-}"
-	DOCKER_GITCOMMIT=""
+	DOCKER_GITCOMMIT="906eacd"
```
Comment (tianon) [inline app-emulation/docker/docker-9999.ebuild]: This should still be empty in the live ebuild. :smile:

File: app-emulation/docker/docker-9999.ebuild [hunk 2]
```diff
@@ -223,10 +221,12 @@ src_compile() {
 }
 
 src_install() {
-	cd "src/${EGO_PN}" || die
 	VERSION="$(cat VERSION)"
 	newbin "bundles/$VERSION/dynbinary-client/docker-$VERSION" docker
 	newbin "bundles/$VERSION/dynbinary-daemon/dockerd-$VERSION" dockerd
+	dosym containerd /usr/bin/docker-containerd
+	dosym containerd-shim /usr/bin/docker-containerd-shim
+	dosym runc /usr/bin/docker-runc
```
Comment (tianon) [inline app-emulation/docker/docker-9999.ebuild]: Nice, seems simple enough!

**Comment by williamh** [comment]
This has been updated. What do you think about the latest changes?

**Comment by tianon** [comment]
Looks good, thanks! :+1:

**Pull request merged**